Apps/Sandcastle/gallery/Image-Based Lighting.html

```diff
@@ -59,14 +59,20 @@
         //Sandcastle_Begin
         var viewer = new Cesium.Viewer("cesiumContainer");
 
+        if (!Cesium.FeatureDetection.supportsBasis(viewer.scene)) {
+          window.alert(
+            "This browser does not support the Basis Universal format"
+          );
+        }
+
         if (!viewer.scene.specularEnvironmentMapsSupported) {
           window.alert(
             "This browser does not support specular environment maps."
           );
         }
 
         var environmentMapURL =
-          "https://cesium.com/public/SandcastleSampleData/kiara_6_afternoon_2k_ibl.ktx";
+          "https://cesium.com/public/SandcastleSampleData/kiara_6_afternoon_2k_ibl.ktx2";
```

Comment:
Do you have all versions of kiara to test with (8-bit unorm, 16-bit float, 32-bit float)? I'm curious to try them all out.